### PR TITLE
Update GenOuest config

### PR DESCRIPTION
**Pull request by abretaud**
Updating the GenOuest profile (added new default mounted volume + updated cpu/mem limits)

### CODE REVIEW

**Comment by abretaud** [comment]
ping @nf-core/configs-team for a little review pleae :)

**Comment by jfy133** [comment]
Sorry @abretaud ! We replaced configs team with nf-core/maintainers  but forgot to update the links here! Looking nnow!

File: docs/genouest.md
Comment (jfy133) [inline docs/genouest.md:14]: Note many pipelines use a more recent minimum version

**Review by jfy133 — APPROVED** [review]
LGTM if no `docs.md` info needs to be updated :)

**Pull request merged**
